gh-pages/Gemfile

```diff
@@ -4,6 +4,9 @@ gem "github-pages", group: :jekyll_plugins
 
 gem "tzinfo-data"
 gem "wdm", "~> 0.1.0" if Gem.win_platform?
+gem "kramdown", "~> 2.3.0"
+gem "activesupport", "~> 4.2.11"
+gem "tzinfo", "~> 1.1"
```

Comment:
Doesn't the latest version work as well?

Reply:
- tzinfo version must be < 2.0 because of package version conflicts.
- activesupport - the latest minor/hotfix version is installed for major verion 4. I did not want to upgrade the major version in this issue.
- kramdown 2.3.0 is the latest one

But I agree to improve kramdown version constraint for future updates.